### PR TITLE
SunPy Project Mission and Vision Statement

### PR DESCRIPTION
**Pull request by ehsteve**
Adding a vision and mission statement. This addresses issue #47. Some initial discussion occurred on this [google doc](https://docs.google.com/document/d/1kYsVqGNMnAj2DCYVi1juC4T3fDk4scAmZoi7v_KTWS0/edit?usp=sharing) which included inputs by at least @DanRyanIrish, @kreardon, @nabobalis, and @wafels.

### CODE REVIEW

**Comment by ehsteve** [comment]
Original discussion and drafting occurred on this [google doc](hhttps://docs.google.com/document/d/1kYsVqGNMnAj2DCYVi1juC4T3fDk4scAmZoi7v_KTWS0/edit).

**Comment by ehsteve** [comment]
This should only be considered a starting point and needs significant input from the community.

File: SEP-0010.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+The vision statement is intended as a guide to help the project make decisions that align with its philosphy.
+A vision statement is meant to be inspirational and idealistic.
+It is meant to describe a future that the Project is trying to make a reality.
+
+The mission statement is a formal summary of the goals of the Project.
+It describes what the Project is trying to achieve.
+It should be consistent with the vision statement.
+
+# Vision Statement
+A diverse and inclusive solar physics and heliophysics community that supports scientific discovery and reproducibility through the development of accessible, open-source software.
```
Comment (nabobalis) [inline SEP-0010.md:27]: Can we combine "solar physics and heliophysics" to have one less physics?

I also feel that starting with A feels a bit passive.

File: SEP-0010.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# Mission Statement
+To develop and facilitate a high quality, easy-to-use, community-led, free and open-source solar data-analysis ecosystem based on the scientific Python environment.
```
Comment (nabobalis) [inline SEP-0010.md:30]: solar or helio or both?

File: SEP-0010.md (hunk 1 above)
Comment (ehsteve) [reply SEP-0010.md:27]: People wanted solar physics mentioned in particular, maybe we just remove heliophysics? But i remember folks saying that we wanted to think more broadly than just solar.

File: SEP-0010.md (hunk 2 above)
Comment (ehsteve) [reply SEP-0010.md:30]: See other comment.

File: SEP-0010.md (hunk 1 above)
Comment (nabobalis) [reply SEP-0010.md:27]: I am pro it being more broad, I wondering if we could combine both into one fewer word.

**Review by rhewett — APPROVED** [review]

**Review by tiagopereira — APPROVED** [review]

File: SEP-0010.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# SEP-0009 -- Release Pattern for SunPy Core
```
Comment (tiagopereira) [inline SEP-0010.md:1]: Should be SEP-0010

File: SEP-0010.md (hunk 1 above)
Comment (Cadair) [inline SEP-0010.md:27]: Something along the lines of this to make it clear it's community developed?
```suggestion
A diverse and inclusive solar physics and heliophysics community that supports scientific discovery and reproducibility through the development of accessible, open-source and open-development software.
```

**Review by binchensun — APPROVED** [review]

**Review by mbobra — APPROVED** [review]

**Review by kreardon — APPROVED** [review]

**Comment by nabobalis** [comment]
Was this meant to be merged with some of the suggestions unresolved, or will we be doing another pass on it soon?

**Pull request merged**